.gitignore

```diff
@@ -20,3 +20,5 @@ out/
 
 # Go best practices dictate that libraries should not include the vendor directory
 vendor
+*.plan
+*.tfvars
```

Comment:
There's a section at the top of the `.gitignore` for Terraform files. Please move these up there, and remove the `terraform.tfvars`, since your rule supersedes it.

Reply:
[Changed with commit db0e58dc](https://github.com/hashicorp/terraform-aws-nomad/pull/10/commits/db0e58dc09506b22d4e83d1ff9f2b2b23146386d)